src/main/kotlin/no/skatteetaten/aurora/boober/feature/azure/AzureFeature.kt

```diff
@@ -36,15 +37,26 @@ class AzureFeature(
     private val auroraApim = AuroraAzureApimSubPart()
 
     override fun isActive(spec: AuroraDeploymentSpec): Boolean {
-        return spec.isJwtToStsConverterEnabled || spec.isAzureAppFqdnEnabled || spec.isApimEnabled
+        return !isAzureSpecificallyDisabled(spec) &&
+            (spec.isJwtToStsConverterEnabled || spec.isAzureAppFqdnEnabled || spec.isApimEnabled)
     }
 
+    private fun isAzureSpecificallyDisabled(spec: AuroraDeploymentSpec): Boolean =
+        spec.getOrNull<String>("azure")?.let { it == "false" } == true
+
     override fun enable(header: AuroraDeploymentSpec): Boolean {
         return !header.isJob
     }
 
     override fun handlers(header: AuroraDeploymentSpec, cmd: AuroraContextCommand): Set<AuroraConfigFieldHandler> {
-        return jwtToStsConverter.handlers(sidecarVersion, defaultLdapUrl, defaultAzureJwks) +
+        return setOf(
+            AuroraConfigFieldHandler(
```

Comment:
Hva skjer hvis noen tar azure: true ? Feiler det kolossalt da?

Reply:
nei, ingenting skjer; det mangler et azureAppFqdn som må fins. Se https://github.com/Skatteetaten/boober/pull/567/files/c09c40ee75d4618ab002bd51f27db8adb55d8171#diff-9ce8fd0d22788dbdedd1d0f42c2649a34d11aba5ae3f5bea86ced3c96c55bcf8R41 for logikken her.